### PR TITLE
Cart

### PR DESCRIPTION
**Pull request by ArtemHurin**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -100,15 +101,22 @@
             <version>2.12.0</version>
             <scope>compile</scope>
         </dependency>
-
+        <!-- https://mvnrepository.com/artifact/org.apache.httpcomponents/httpclient -->
```
Comment (ostret) [inline pom.xml:104]: where do you use those ?

File: src/main/java/pages/cart/ShoppingItem.java [hunk 2]
```diff
@@ -168,6 +157,18 @@ public String getNameText () {
        return getName().getText();
     }
 
+    /**
+     * decrease quantity value
+     * using explicitly wait to check the total price
+     */
+    @Step("Click down arrow to decrease quantity of product")
+    public void changeValueByDecreasing() {
+        OurWebDriver.driver.manage().timeouts().implicitlyWait(0, TimeUnit.SECONDS);
```
Comment (ostret) [inline src/main/java/pages/cart/ShoppingItem.java:166]: Maybe there's better way to wait for it?

File: src/main/java/test/artem/ArtemAPIGetTest.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+    @Severity(SeverityLevel.CRITICAL)
+    @Description("Verification of presents several item in chosen cart")
+    @Test
+    public void getCartItemTest() {
+        //Arrange
+        RestAssured.baseURI = EnviromentHelper.getPrestaUrl();
+        String expected = "5";
+        //Act
+        RestAssured
```
Comment (ostret) [inline src/main/java/test/artem/ArtemAPIGetTest.java:25]: this test doesn't make much sense to be honest 
in real life scenario this won't find any defects

**Review by ostret — COMMENTED** [review]
try to keep dependencies clean and think about what you are actually testing with API tests

**Review by SET-SKYNET — APPROVED** [review]

**Review by ShBr2k — APPROVED** [review]
Done.

File: pom.xml (hunk 1 above)
Comment (ArtemHurin) [reply pom.xml:104]: I used it on my first try of writing API tests and forgot to delete, I will fix it

File: src/main/java/pages/cart/ShoppingItem.java (hunk 2 above)
Comment (ArtemHurin) [reply src/main/java/pages/cart/ShoppingItem.java:166]: It works only with explicity wait, implicity wait doesn`t enough

File: src/main/java/test/artem/ArtemAPIGetTest.java (hunk 3 above)
Comment (ArtemHurin) [reply src/main/java/test/artem/ArtemAPIGetTest.java:25]: it`s simple get test to check that we can get response from our lovely prestashop, my main test it`s post test

**Pull request closed without merging**